src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -101,6 +101,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     UUID conversationId = conversation.getId();
 
     List<Message> messages = messageStore.getMessagesInConversation(conversationId);
+    
+
+    String username = (String) request.getSession().getAttribute("user");
+    if (username != null) {
+      User user = userStore.getUser(username);
+      if (user != null) {
+        request.setAttribute("user", user);
```

Comment:
Why does the request need this attribute? Can't you just get it from the session?